wagtail/users/tests/test_admin_views.py

```diff
@@ -1806,3 +1806,257 @@ def test_get_group_viewset_cls_custom_form_does_not_exist(self):
                 'Module "wagtail.users.tests" does not define a "CustomClassDoesNotExist" attribute/class',
                 str(exc_info.exception),
             )
+
+
+class TestAuthorizationIndexView(TestCase, WagtailTestUtils):
```

Comment:
```suggestion
class TestAuthorisationIndexView(TestCase, WagtailTestUtils):
```

Small nitpick but we do try to use English UK (not US) spelling when in the code or documentation.

Note, if referring to something like [`HTTP Authorization`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization) we would use US spelling but a variable or test case should use the UK spelling.

https://docs.wagtail.org/en/latest/contributing/general_guidelines.html#language

Reply:
Got it! Thanks for letting me know.